kdl_parser/package.xml

```diff
@@ -27,11 +27,12 @@
   <build_depend>roscpp</build_depend>
   <build_depend>urdf</build_depend>
   <build_depend>cmake_modules</build_depend>
-  <build_depend>rostest</build_depend>
 
-  <run_depend version_gte="1.3.0">orocos_kdl</run_depend>
-  <run_depend>rosconsole</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>urdf</run_depend>
+  <exec_depend version_gte="1.3.0">orocos_kdl</exec_depend>
+  <exec_depend>rosconsole</exec_depend>
+  <exec_depend>roscpp</exec_depend>
+  <exec_depend>urdf</exec_depend>
```

Comment:
Add `<build_export_depend>` with `urdf`, `roscpp`, `tinyxml`, and `orocos_kdl` since they're in an installed header?

https://github.com/ros/kdl_parser/blob/71e728edffa4d41cfbd3b44839a3bfd7d04aa368/kdl_parser/include/kdl_parser/kdl_parser.hpp#L40-L43